.gitignore

```diff
@@ -2,3 +2,4 @@ target/
 bcc-sys/src/bccapi.rs
 Cargo.lock
 *.swp
+.idea
```

Comment:
Let's not add ignores that are specific to your development environment. Those should be added in your local gitignore (~/.gitignore) only - we don't want to cover every possibility here.

Reply:
Good point! Removed!